core/Cargo.toml

```diff
@@ -328,6 +329,8 @@ suppaftp = { version = "5.2", default-features = false, features = [
 ], optional = true }
 # for services-tikv
 tikv-client = { version = "0.3.0", optional = true, default-features = false }
+# for services-hdfs-native
+hdfs-native = { version = "0.6.0"}
```

Comment:
Please add `optional = true` and enable it in `services-hdfs-native` so we won't enable this feature by default.